libr/anal/p/anal_sbpf.c

```diff
@@ -487,11 +498,34 @@ static bool sbpf_analyze_strings(RAnal *anal) {
 
 				// Create flag with format: ptr.<pointer_addr>_<string>
 				snprintf (flagname, sizeof (flagname), "ptr.%"PFMT64x"_%s", ref->addr, safe_str);
+
+				// Save string for comment (unfiltered for readability)
+				ut32 comment_len = size < 250 ? size : 250;
```

Comment:
move this logic into a separate function to make it more readable and reduce locs